dashboard/views.py

```diff
@@ -643,8 +644,8 @@ def get_course_assignments(course_id):
     assignments_in_course[['points_possible','group_points']]=assignments_in_course[['points_possible','group_points']].fillna(0)
     assignments_in_course[['points_possible', 'group_points','weight']] = assignments_in_course[['points_possible', 'group_points','weight']].astype(float)
     consider_weight=is_weight_considered(course_id)
-    df2 = assignments_in_course[['weight','group_points','grp_id']].drop_duplicates()
-    hidden_assignments = are_weighted_assignments_hidden(course_id, df2)
+    df_assignment = assignments_in_course[['weight','group_points','grp_id']].drop_duplicates().copy()
```

Comment:
is this `df_assignment` or `df_greaterthan_percent`?

Reply:
This is in another function get_course_assignments, just another instance of "df2" but totally unrelated to the previous example.